cmd/cloud/deployment.go

```diff
@@ -307,8 +307,8 @@ func deploymentUpdate(cmd *cobra.Command, args []string) error {
 		return errors.Wrap(err, "failed to find a valid workspace")
 	}
 
-	if dagDeployEnable != "" && !(dagDeployEnable == enable || dagDeployEnable == disable) {
-		return errors.New("Invalid --dag-deploy value. Possible values are (enable, disable)")
+	if dagDeploy != "" && !(dagDeploy == enable || dagDeploy == disable) {
+		return errors.New("Invalid --dag-deploy value)")
```

Comment:
nitpick: might want to add unit tests for this case

Reply:
Done